icaruscode/Timing/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ set(	LIB_LIBRARIES
         wda::wda
         icaruscode_IcarusObj
         sbnobj::Common_PMT_Data
+	larevt::CalibrationDBI_IOVData
```

Comment:
Is the dependency on `wda::wda` above still needed?

Reply:
Not anymore. I already removed the `#include` statements, but I forgot to remove it from `CMakeLists.txt`.